src/dask_histogram/core.py

```diff
@@ -63,7 +63,7 @@ def _blocked_sa(
     thehist = (
         clone(histref)
         if not isinstance(histref, tuple)
-        else bh.Histogram(*histref[0], storage=histref[1], metadata=histref[2])
+        else bh.Histogram(*histref[0], storage=histref[1](), metadata=histref[2])
```

Comment:
`storage=histref[1]()`
The signature of histref changed so this wasn't called there but does get called here. Why is this?

Reply:
The inputs to the histgrams are the instances, but the `histref`/`_meta` is stored as the type now.

It was inconsistent before. I can put if back if you like that better?

Reply:
I don't mind which way around it is, but this caused a lot of line changes that were confusing and didn't seem to do much.

Reply:
The doctest was failing and it led to some slinky-straightening.